streams/src/test/java/org/apache/kafka/streams/kstream/RepartitionTopicNamingTest.java

```diff
@@ -522,182 +521,181 @@ public void process(final String key, final String value) {
-            "    Source: KSTREAM-SOURCE-0000000026 (topics: [reduced-stream-repartition])\n" +
-            "      --> KSTREAM-REDUCE-0000000023\n" +
-            "    Processor: KSTREAM-REDUCE-0000000023 (stores: [KSTREAM-REDUCE-STATE-STORE-0000000022])\n" +
-            "      --> KTABLE-TOSTREAM-0000000027\n" +
-            "      <-- KSTREAM-SOURCE-0000000026\n" +
-            "    Processor: KTABLE-TOSTREAM-0000000027 (stores: [])\n" +
-            "      --> KSTREAM-SINK-0000000028\n" +
-            "      <-- KSTREAM-REDUCE-0000000023\n" +
-            "    Sink: KSTREAM-SINK-0000000028 (topic: outputTopic_2)\n" +
-            "      <-- KTABLE-TOSTREAM-0000000027\n\n";
-
+        "   Sub-topology: 0\n" +
+        "    Source: KSTREAM-SOURCE-0000000000 (topics: [input])\n" +
+        "      --> KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-MAP-0000000001 (stores: [])\n" +
+        "      --> KSTREAM-FILTER-0000000029, KSTREAM-FILTER-0000000002, KSTREAM-FILTER-0000000020, aggregate-stream-repartition-filter, count-stream-repartition-filter\n" +
+        "      <-- KSTREAM-SOURCE-0000000000\n" +
+        "    Processor: KSTREAM-FILTER-0000000020 (stores: [])\n" +
+        "      --> KSTREAM-PEEK-0000000021\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-FILTER-0000000002 (stores: [])\n" +
+        "      --> KSTREAM-MAPVALUES-0000000003\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-FILTER-0000000029 (stores: [])\n" +
+        "      --> joined-stream-left-repartition-filter\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: KSTREAM-PEEK-0000000021 (stores: [])\n" +
+        "      --> reduced-stream-repartition-filter\n" +
+        "      <-- KSTREAM-FILTER-0000000020\n" +
+        "    Processor: KSTREAM-MAPVALUES-0000000003 (stores: [])\n" +
+        "      --> KSTREAM-PROCESSOR-0000000004\n" +
+        "      <-- KSTREAM-FILTER-0000000002\n" +
+        "    Processor: aggregate-stream-repartition-filter (stores: [])\n" +
+        "      --> aggregate-stream-repartition-sink\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: count-stream-repartition-filter (stores: [])\n" +
+        "      --> count-stream-repartition-sink\n" +
+        "      <-- KSTREAM-MAP-0000000001\n" +
+        "    Processor: joined-stream-left-repartition-filter (stores: [])\n" +
+        "      --> joined-stream-left-repartition-sink\n" +
+        "      <-- KSTREAM-FILTER-0000000029\n" +
+        "    Processor: reduced-stream-repartition-filter (stores: [])\n" +
+        "      --> reduced-stream-repartition-sink\n" +
+        "      <-- KSTREAM-PEEK-0000000021\n" +
+        "    Processor: KSTREAM-PROCESSOR-0000000004 (stores: [])\n" +
+        "      --> none\n" +
+        "      <-- KSTREAM-MAPVALUES-0000000003\n" +
+        "    Sink: aggregate-stream-repartition-sink (topic: aggregate-stream-repartition)\n" +
```

Comment:
The order here changes as we sort the names when rendering the topology description, the order of the processors in the physical plan remain the same